### PR TITLE
any member of an operation should see who created the operation

### LINKED ISSUE #2005: develop: Crash for viewing description for operations without admin rights
This affects only the develop branch.
```
Traceback (most recent call last):
  File "/home/icg173/src/mss/mslib/msui/mscolab.py", line 527, in view_description
    creator_name = _json["username"]
KeyError: 'username'
CRITICAL: MSS Version: 8.2.0
CRITICAL: Python Version: 3.10.11 | packaged by conda-forge | (main, May 10 2023, 18:58:44) [GCC 11.3.0]
CRITICAL: Platform: Linux-6.1.0-10-amd64-x86_64-with-glibc2.36 (('64bit', 'ELF'))
CRITICAL: Fatal error: Traceback (most recent call last):
  File "/home/icg173/src/mss/mslib/msui/mscolab.py", line 527, in view_description
    creator_name = _json["username"]
```

### PR DESCRIPTION
**Pull request by ReimarBauer**
**Purpose of PR?**:

Fixes #2005

removing the limitation to the creator to see who created an operation.

### CODE REVIEW

File: mslib/mscolab/file_manager.py [hunk 1]
```diff
@@ -495,7 +495,9 @@ def fetch_users_with_permission(self, op_id, u_id):
         return users
 
     def fetch_operation_creator(self, op_id, u_id):
-        if not self.is_admin(u_id, op_id) and not self.is_creator(u_id, op_id):
+        if (not self.is_admin(u_id, op_id) and not self.is_creator(u_id, op_id) and
+                not self.is_collaborator(u_id, op_id) and not self.is_viewer(u_id, op_id)):
```
Comment (matrss) [inline mslib/mscolab/file_manager.py]: The predicate could be simplified a bit by extracting the not from the parentheses. I.e. `not a and not b` is equivalent to `not (a or b)`. But that is just a nitpick.

File: tests/_test_mscolab/test_file_manager.py [hunk 2]
```diff
@@ -97,8 +97,15 @@ def test_modify_user_special_cases(self):
     def test_fetch_operation_creator(self):
         with self.app.test_client():
             flight_path, operation = self._create_operation(flight_path="more_than_one")
+            self.fm.add_bulk_permission(operation.id, self.user, [self.collaboratoruser.id], "collaborator")
+            self.fm.add_bulk_permission(operation.id, self.user, [self.vieweruser.id], "viewer")
+
             assert operation.path == flight_path
             assert self.fm.fetch_operation_creator(operation.id, self.user.id) == self.user.username
+            assert self.fm.fetch_operation_creator(operation.id, self.collaboratoruser.id) == self.user.username
+            assert self.fm.fetch_operation_creator(operation.id, self.vieweruser.id) == self.user.username
```
Comment (matrss) [inline tests/_test_mscolab/test_file_manager.py:107]: Should there be a fourth assert for `self.adminuser`?

**Review by matrss — COMMENTED** [review]
I like that you have modified the test accordingly as well :)

File: mslib/mscolab/file_manager.py (hunk 1 above)
Comment (ReimarBauer) [reply mslib/mscolab/file_manager.py]: we have also a is_member, which I forgot

**Review by matrss — APPROVED** [review]

**Pull request merged**